config/helm/chart/default/tests/Common/activegate/serviceaccount-activegate_test.yaml

```diff
@@ -0,0 +1,15 @@
+suite: test serviceaccount for activegate
+templates:
+  - Common/activegate/serviceaccount-activegate.yaml
+tests:
+  - it: should exist
+    set:
+      rbac.activeGate.annotations:
```

Comment:
missing a test for kubernetes monitoring service account

Reply:
nice catch 👍

Reply:
Done